internal/wiz/structs.go

```diff
@@ -229,6 +229,33 @@ type SAMLGroupMappingCreateInput struct {
 	Projects        []string `json:"projects"`
 }
 
+// UpdateSAMLGroupMappingPayload struct
+type UpdateSAMLGroupMappingPayload struct {
+	SAMLGroupMapping SAMLGroupMapping `json:"samlGroupMapping,omitempty"`
+}
+
+// UpdateSAMLGroupMappingInput struct
+type UpdateSAMLGroupMappingInput struct {
+	ID    string                       `json:"id"`
+	Patch UpdateSAMLGroupMappingUpsert `json:"patch"`
+}
+
+// UpdateSAMLGroupMappingUpsert struct
+type UpdateSAMLGroupMappingUpsert struct {
```

Comment:
correct type should be `ModifySAMLGroupMappingPatch`. The structs file is meant to mirror the vendor defined schema (as far as possible). Where there are custom structs This should also cover the `DeleteSAMLGroupMapping` custom struct below which should not then be needed.

If there are custom structs, they should go into the provider resource file instead.